lib/LaTeXML/Engine/TeX_Fonts.pool.ltxml

```diff
@@ -285,5 +285,38 @@ DeclareFontMap('OMX',
     #                        [missing tips for horizontal curly braces]
     "\x{2191}", "\x{2193}", undef, undef, undef, undef, "\x{21D1}", "\x{21D3}"]);
 
+#%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+# TeX's ligatures handled by rewrite regexps.
+# Note: applied in reverse order of definition (latest defined applied first!)
+# Note also, these area only applied in text content, not in attributes!
+sub nonTypewriter {
+  my ($font) = @_;
+  return ($font->getFamily ne 'typewriter'); }
+
+sub nonTypewriterT1 {
+  my ($font) = @_;
+  return ($font->getFamily ne 'typewriter') && (($font->getEncoding || 'OT1') =~ /^(OT1|T1)$/); }
+
+# EN DASH (NOTE: With digits before & aft => \N{FIGURE DASH})
+DefLigature(qr{--},  "\x{2013}", fontTest => \&nonTypewriter);    # EN dash
+DefLigature(qr{---}, "\x{2014}", fontTest => \&nonTypewriter);    # EM dash
+
+# Ligatures for doubled single left & right quotes to convert to double quotes
+# [should ligatures be part of a font, in the first place? (it is in TeX!)
+DefLigature(qr{\x{2018}\x{2018}}, "\x{201C}", fontTest => \&nonTypewriterT1);   # double left quote
+DefLigature(qr{\x{2019}\x{2019}}, "\x{201D}", fontTest => \&nonTypewriterT1);   # double right quote
+DefLigature(qr{\?\x{2018}},       UTF(0xBF),  fontTest => \&nonTypewriterT1);   # ? backquote
+DefLigature(qr{!\x{2018}},        UTF(0xA1),  fontTest => \&nonTypewriterT1);   # ! backquote
+# These ligatures are also handled by TeX.
+# However, it appears that decent modern fonts in modern browsers handle these at that level.
+# So it's likely not worth doing it at the conversion level, possibly adversely affecting search.
+# DefLigature(qr{ff},               "\x{FB00}", fontTest => \&nonTypewriterT1);
+# DefLigature(qr{fi},               "\x{FB01}", fontTest => \&nonTypewriterT1);
+# DefLigature(qr{fl},               "\x{FB02}", fontTest => \&nonTypewriterT1);
+# DefLigature(qr{ffi},              "\x{FB03}", fontTest => \&nonTypewriterT1);
+# DefLigature(qr{ffl},              "\x{FB04}", fontTest => \&nonTypewriterT1);
+
+DefLigature(qr{\.\.\.}, "\x{2026}", fontTest => sub { $_[0]->getFamily ne 'typewriter'; });  # ldots
```

Comment:
this test can also be `\&nonTypewriter`

Reply:
Ah the benefits of good modularity! will do